lib/syscall_shim/uk_syscall_binary.c

```diff
@@ -116,6 +116,12 @@ void ukplat_syscall_handler(struct uk_syscall_ctx *usc)
 		    execenv->regs.__syscall_rarg1);
 #endif /* CONFIG_LIBSYSCALL_SHIM_DEBUG_HANDLER */
 
+	/* Mark system call entry */
```

Comment:
I'm clearly missing something here... What is the reason we don't call the wrapper functions (and hence the check for nested syscalls) in binary syscalls?

Reply:
hmm looks like i forgot to replace this place with the new wrapper. Thanks for catching that!